CHAP/runner.py

```diff
@@ -109,6 +103,13 @@ def run(pipeline_config, interactive=False, logger=None, log_level=None, log_han
         else:
             name = item
         if "users" in name:
+            # load user processor
+            try:
+                import users
+            except ImportError:
+                if logger:
+                    logger.error(f'Unable to load {name}')
+                continue
```

Comment:
Sorry, I'm still unclear as to why users needs to be imported as a module at all. The submodule of users containing the specific UserProcessor needed is imported on L115 (with `module = __import__(modName, fromlist=[clsName])`) -- am I missing something?

Reply:
Ah, I see now. Thanks for clarifying